satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -333,4 +333,23 @@ read all (
     select api_key
     where api_key.project_id = ?
     orderby asc api_key.name
-)
\ No newline at end of file
+)
+
+//--- certDB ---//
```

Comment:
If you rename this to just 'cert' or 'certRecord',  I think it will clear up some of the naming weirdness ala 'Create_CertDB()'

Reply:
done.. made certRecord... seems more meaningful :-)